### DISCUSSION TITLE
MUSHR Support

### DISCUSSION THREAD

**Discussion by chuckl72** [prl-mushr/mushr#111] [General]
I'm considering put together the MUSHR car and want to make sure that there is support.  Looking at the dates of the post, it seems like there hasn't been much activity in recent years.  My big question, do the software packages still work?  I've had issues where the updated packages on project like this don't work.  Any comments would be appreciated.
Chuck

**Comment by schmittlema** [comment]
Yes it is being actively supported, though we are a small team. I would recommend starting with the [noetic install](https://mushr.io/tutorials/noetic_quickstart/) as that is what we are actively supporting
